src/events/utils.py

```diff
@@ -7,11 +7,17 @@
     from events.models import EventPage
 
 
-def get_event_date(event: "EventPage") -> str:
-    return event.event_date.strftime("%-d %B %Y")
+def get_event_start_date(event: "EventPage") -> str:
+    return f"{event.event_start.strftime("%-d %B %Y")}"
```

Comment:
This doesn't actually need to be a f-string formatter
```suggestion
    return event.event_start.strftime("%-d %B %Y")
```